fe/fe-core/src/main/java/com/starrocks/task/UpdateTabletMetaInfoTask.java

```diff
@@ -101,6 +106,22 @@ public UpdateTabletMetaInfoTask(long backendId,
         this.latch = latch;
     }
 
+    public UpdateTabletMetaInfoTask(long backendId,
+                                    Set<Pair<Long, Integer>> tableIdWithSchemaHash,
+                                    boolean metaValue,
+                                    MarkedCountDownLatch<Long, Set<Pair<Long, Integer>>> latch,
+                                    TTabletMetaType metaType, TTabletType tabletType, long txnId) {
+        this(backendId, tableIdWithSchemaHash, metaType);
+        if (metaType == TTabletMetaType.INMEMORY) {
+            this.isInMemory = metaValue;
+        } else if (metaType == TTabletMetaType.ENABLE_PERSISTENT_INDEX) {
+            this.enablePersistentIndex = metaValue;
+        }
```

Comment:
else?

Reply:
Reuse the previous constructor, I think it can be guaranteed outside